### DISCUSSION TITLE
Polygon Annotation

### DISCUSSION THREAD

**Discussion by cmalai** [kinhong/OpenLabeler#21] [General]
Hi Kinhong.

Thanks for this amazing good OpenLabeler. We are an upcoming EduTech company supporting unemployed graduates to upskill [for free] on skills which are required today. I was trying to you use your tool for labeling some images and teach annotation, and realised the rectangle box based labeling works perfect, but i am unable to finish the polygon shape.

i start with a double click, then use single clicks to mark the points, then i assumed i have to double click on the point where i need to finish the polygon, but it doesnt seem to stop. So we are unable to complete our polygon annotation.

![image](https://user-images.githubusercontent.com/2596136/169635700-5e62c571-e571-4aa8-815d-5ee15089c2f8.png)

**Comment by kinhong** [comment]
Hi @cmalai , this issue should be fixed in v1.4.1.
